org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/entities/participants/EntitiesCompletionParticipant.java

```diff
@@ -67,7 +68,7 @@ public void onXMLContent(ICompletionRequest request, ICompletionResponse respons
 	 * @param request     the completion request.

```

Comment:
`* @param request     the completion request.` can be removed since `request` is not a parameter for this method.

Reply:
fixed